src/components/UNSTABLE__Pagination/__tests__/PageSelector.spec.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file Page selector tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { PageSelector } from '../../../..';
+
+import { namespace } from '../PageSelector';
+
+describe('PageSelector', () => {
+  test('should have root class with security namespace', () => {
```

Comment:
Would you mind elaborating on the purpose of this test?

Reply:
I elaborated on it earlier before I made the change & I believe you said you liked the idea: https://github.com/carbon-design-system/ibm-security/pull/327#discussion_r373153604

> Though on the topic of multiple tests files, I am going to make a new file for a couple PageSelector tests, I think. At least to check for a namespace class, since that class is relied on so heavily for styles not breaking etc.

I'm not sure what more can be said about it. The point about the importance of the class, and making sure it doesn't get deleted due to how heavily it is relied on for styles, is the main/only thing.

Reply:
Apologies for any confusion or misdirection - I like the idea of breaking out the `PageSelector` component and testing the behavior and logic of that, but I don't believe this kind of test is entirely necessary.

Reply:
Okay... unfortunately I don't see any "behavior" tests for the `PageSelector` to break out of the other test file. Every test in the main `Pagination` file is related to the pagination component's behavior, as a whole. And the `PageSelector` is just a Carbon `Select`, which they test on their end. So I will delete this file.